<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestFluorescentFlatClass</class>
 <widget class="QMainWindow" name="TestFluorescentFlatClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1173</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestFluorescentFlat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1161</width>
      <height>261</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>0</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>０</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>１</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>２</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>３</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>４</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>５</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>６</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>７</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>８</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>９</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>１０</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>１１</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>１２</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>１３</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>１４</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>１５</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>１６</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>１７</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>１８</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>１９</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>２０</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>２１</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>２２</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>２３</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>２４</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>２５</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>２６</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>２７</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>２８</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>２９</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButtonOK">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>270</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Transfer</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxSetAll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2047</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSetAll">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set All</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonOnOff">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ON/OFF</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
